.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,7 @@
 blank_issues_enabled: false
```

Comment:
`.github/ISSUE_TEMPLATE/config.yml` and `.github/ISSUE_TEMPLATE/FEATURE.yml` was added before but because repo was not public was not visible. Now this feature is active and I just adjusted the link to the supported boundaries.